cmd/access/staked_access_node_builder.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"github.com/onflow/flow-go/cmd"
+)
+
+type StakedAccessNodeBuilder struct {
+	*FlowAccessNodeBuilder
+}
+
+func StakedAccessNode(anb *FlowAccessNodeBuilder) *StakedAccessNodeBuilder {
```

Comment:
wanted to avoid as much as possible the `if staked then else ` paradigm hence added two new sub node builder sub classes.